compiler/daml-lf-reader/src/DA/Daml/LF/Reader.hs

```diff
@@ -29,6 +30,16 @@ lineToKeyValue line = case splitOn ":" line of
     [l, r] -> (trim l , trim r)
     _ -> error $ "Expected two fields in line " <> line
 
+appendToFirstEntry :: [String] -> String -> [String]
+appendToFirstEntry (h : t) nextLine = (h ++ nextLine) : t
+appendToFirstEntry _ _ = error $ "Reading Manifest file from dar failed."
+
+multiLineContent :: [String] -> [String] -> [String]
```

Comment:
The specification reads if a line exceeds 72 bytes start with a space on the next line, collecting lines with space and appending to the previous line seems very easy to implement. Let me know if you think that is too hacky